### ISSUE TITLE
This still loads a tracking script

### ISSUE THREAD

**Issue by slang800** [Esteban-Rocha/page-ruler-redux#9]
https://github.com/Esteban-Rocha/page-ruler-redux/blob/master/src/background.js#L314

Google Analytics is hardly better than mixpanel.

**Comment by Esteban-Rocha** [comment]
https://twitter.com/random_walker/status/960536723095719937

**Comment by Esteban-Rocha** [comment]
The idea is to have someway of telemetry to improve the app, I may suggest adding an opt out option for this.

**Comment by slang800** [comment]
Yeah, I read the tweet. My point is that this fork is advertised as "100% free", "open source" and having "no adware or malware". Including a proprietary script that invades the privacy of your users hardly seems to fit with the spirit of that description.

Do you even use the data that you collect to make decisions, or are you just trying to collect as many metrics as possible?

**Comment by Esteban-Rocha** [comment]
LOL I haven't even looked at that in a long time as I have been quite busy with work, But I think that if you're so worried with GA scripts, using this extension should be your last of concerns.

**Comment by slang800** [comment]
> using this extension should be your last of concerns

huh? what else do you think I should be concerned about?

**Comment by Esteban-Rocha** [comment]
Hi @slang800, how's your day? I have to say that I disagree on your views...

> huh? what else do you think I should be concerned about?

Using any website whatsoever? All internet is tracking, even using chrome is a suicide if you're going to be as picky to put GA as a greater evil than what mixpanel was doing that was stealing passwords...

> Including a proprietary script that invades the privacy of your users hardly seems to fit with the spirit of that description.

I dont think so, it is common practice to use Google analytics to get insights of pageviews and useful info on websites effectiveness, in this case I'm not refusing to drop it, as I clearly stated above I can make that available only for those who want to share stats to opt-in, so far I haven't even did anything yet with that because this was implemented in the original extension and I just swapped the original UA ID, I haven't come into that part yet to even do something or implement telemetry these are just random ideas that may be good for improving the ext.

I really sorry but I do not think your privacy concerns are "truthy" when in your personal website first thing you have is your personal GA ID "UA-42745912-1" loaded, which means by your own definition you're "invading the privacy of your website viewers".. Yet you only come to remark how untrusty I\this ext is for having a script that counts page views.. if GA is not set right it hardly get anything more than that, and it is not.

So yes... I'm totally open for feedback, if people feel uncomfortable with that GA which just counts installs I'll wipe it no problem, just keep it cool and honest.. No need to point fingers when your doing exactly the thing your condemning.

**Comment by slang800** [comment]
> Using any website whatsoever? All internet is tracking, even using chrome is a suicide if you're going to be as picky to put GA as a greater evil than what mixpanel was doing that was stealing passwords...

I wouldn't put much value on "someone else is doing it therefore it doesn't matter if I do it" argument, but I should clarify a few things:

- I rarely load tracking scripts from websites. I block most JS. Server-side tracking doesn't bother me as much, but I usually route my traffic through Tor anyway.
- I don't use Google-branded Chrome, my browser is built from the open-source [Chromium project](https://www.chromium.org/).
- I'm not saying that GA is "a greater evil" than mixpanel. I don't care which is worse. I'm saying that switching to Google Analytics is hardly better than mixpanel.

> I haven't come into that part yet to even do something ... if GA is not set right it hardly get anything more than that, and it is not.

If you're not using it, then why bother collecting the data in the first place? That just sounds like a waste of bandwidth.

> your personal website first thing you have is your personal GA ID ... by your own definition you're "invading the privacy of your website viewers".

I don't advertise http://slang.cx as "100% free", "open source" and having "no adware or malware". I _am_ invading your privacy and I hope that bothers you. It's not "truthy", it's hypocritical.
